jupyterhub/tests/selenium/test_browser.py

```diff
@@ -519,85 +510,27 @@ async def open_token_page(app, browser, user):
     await asyncio.sleep(0.1)
 
 
-def elements_API_tokens_table(browser):
-    """function to get table of tokens as element
-    and getting the attributes for each tokens"""
-    try:
-        return browser.find_element(*TokenPageLocators.TABLE_API)
-    except NoSuchElementException:
-        return WebDriverWait(browser, 10).until(
-            EC.presence_of_element_located(TokenPageLocators.TABLE_API)
-        )
-
-
-def elements_API_tokens_table_body(browser):
-    """function to get body table of tokens as element
-    and getting the attributes for each tokens"""
-    try:
-        table = elements_API_tokens_table(browser)
-        return table.find_element(*TokenPageLocators.TABLE_API_BODY)
-    except NoSuchElementException:
-        return WebDriverWait(browser, 10).until(
-            EC.presence_of_element_located(TokenPageLocators.TABLE_API_BODY)
-        )
-
-
-def elements_API_tokens_row_body(browser):
-    """function to get table rows of tokens as elements
-    and getting the attributes for each tokens"""
-    try:
-        body = elements_API_tokens_table_body(browser)
-        return body.find_elements(*TokenPageLocators.TABLE_API_ROWS_BY_CLASS)
-    except NoSuchElementException:
-        return WebDriverWait(browser, 10).until(
-            EC.presence_of_all_elements_located(
-                TokenPageLocators.TABLE_API_ROWS_BY_CLASS
-            )
-        )
+def token_table_body_as_dict(browser, table_locator):
+    # get the token table elements (excluding of the tables header)
+    table = browser.find_element(*table_locator)
+    body = table.find_element(By.TAG_NAME, 'tbody')
+    row = body.find_elements(By.CLASS_NAME, 'token-row')
+    # return dict contains elements in row and column
+    return [
+        [element.text for element in row.find_elements(By.TAG_NAME, 'td')]
+        for row in body.find_elements(By.CLASS_NAME, 'token-row')
```

Comment:
Should this be:

```suggestion
        for row in body.find_elements(By.TAG_NAME, 'tr')
```

The `row` above  on L517 also looks to be unused.